DigitalLearningSolutions.Data/Models/Centre.cs

```diff
@@ -31,8 +31,11 @@ public class Centre
         public int TrainerSpots { get; set; }
         public string? IpPrefix { get; set; }
         public string? ContractType { get; set; }
+        public int CentreTypeId { get; set; }
+        public string CentreType { get; set; }

```

Comment:
This is breaking c#'s null checks, because `CentresDataService.GetCentreDetailsById()` returns a `Centre?` but doesn't populate these fields.
And there are other nullable fields which don't get populated by the new `GetAllCentreSummaries()`.

It's a tricky one to navigate, because we might end up with 5 or 6 models which have centre properties and which overlap in different ways. It's not ideal to have a separate class for each if you're changing some property names. But maybe it's better than this? The only other thing I can think of is having an interface for each one, which are all implemented by `Centre`. What are your thoughts?

Reply:
I wasn't sure on this one.
I've experienced the mutli-overlapping model approach before and see it get out of hand. Although a strict oversight should keep it in check.
One thing I was thinking about but needs discussing is the use of a base class which holds the core properties and then other variations extend from that. But where do you draw the line on what are the core properties?

Reply:
We've decided to use a new model for this instead of expanding Centre.

Reply:
Done.